src/rws_client.cpp

```diff
@@ -136,6 +136,28 @@ RWSClient::RWSResult RWSClient::getIOSignal(const std::string& iosignal)
   return evaluatePOCOResult(httpGet(uri), evaluation_conditions);
 }
 
+RWSClient::RWSResult RWSClient::getMechanicalUnitStaticInfo(const std::string& mechunit)
+{
+  std::string uri = generateMechanicalUnitPath(mechunit) + "?resource=static";
+
+  EvaluationConditions evaluation_conditions;
+  evaluation_conditions.parse_message_into_xml = true;
+  evaluation_conditions.accepted_outcomes.push_back(HTTPResponse::HTTP_OK);
+
+  return evaluatePOCOResult(httpGet(uri), evaluation_conditions);
```

Comment:
Can `httpGet(..)` fail? If so: how is that communicated to the user?

Reply:
See https://github.com/ros-industrial/abb_librws/pull/105#discussion_r511763076